sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -139,17 +139,21 @@ class MapboxMap :
    *
    * @param styleUri The style URI
    * @param styleTransitionOptions style transition options applied when loading the style
+   * @param earlyStyleCallback whether to invoke [onStyleLoaded] callback earlier on the `StyleDataLoaded#Style`]
+   *    event, default to false. Enable earlyStyleCallback helps with getting the Style Object faster,
+   *    so that the runtime styling can be applied immediately after the style is parsed.
    * @param onStyleLoaded The OnStyleLoaded callback
    * @param onMapLoadErrorListener The OnMapLoadErrorListener callback
    */
   fun loadStyleUri(
     styleUri: String,
     styleTransitionOptions: TransitionOptions? = null,
+    earlyStyleCallback: Boolean = false,
```

Comment:
not sure `earlyStyleCallback` is good and descriptive naming but can't think of anything better for now..

Reply:
Any suggestion is welcome, that's the most relevant name I came up with so far..

Reply:
maybe `enableQuickStyleLoad` or `quickStyleLoad` 🤔

Reply:
Please check https://github.com/mapbox/mapbox-maps-android/pull/1353#issuecomment-1127501925 , we have removed this boolean parameter and used a separate `OnStylePreloaded` listener instead. Please provide feedback on the new listener's name.